indy_common/test/auth/test_auth_strategies.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from indy_common.auth_cons_strategies import LocalAuthStrategy
+from indy_common.auth_constraints import AuthConstraint
+
+
+@pytest.fixture(scope='function')
+def auth_map(action_add, action_edit):
+    """
+    action_ids:
+    for add: "ADD--SomeType--some_field--*--new_value"
+    for edit: "EDIT--SomeType--some_field--old_value--new_value"
+    """
+    return {action_add.get_action_id(): AuthConstraint(role="Actor",
+                                                       sig_count=3),
+            action_edit.get_action_id(): AuthConstraint(role="Actor",
+                                                        sig_count=2)}
+
+
+@pytest.fixture(scope='function')
+def local_auth_strategy(auth_map):
+    return LocalAuthStrategy(auth_map=auth_map)
+
+
+def test_local_strategy_found_the_same_action_id(local_auth_strategy):
+    assert local_auth_strategy.get_auth_constraint("ADD--SomeType--some_field--*--new_value")
```

Comment:
Please check that expected constraint is found. Please add more tests (is_accepted_action_id ?) for different cases, especially with *.

Reply:
done